From 26216bac2a3750fc2fdb515deb14c96869608650 Mon Sep 17 00:00:00 2001
From: Matthew Toohey <contact@mtoohey.com>
Date: Sat, 9 Dec 2023 15:33:53 -0500
Subject: [PATCH] Vendor mountless modules

---
 modules/client.go | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/modules/client.go b/modules/client.go
index b9a2a48d4..cd1f18780 100644
--- a/modules/client.go
+++ b/modules/client.go
@@ -235,12 +235,6 @@ func (c *Client) Vendor() error {
 			continue
 		}
 
-		// See https://github.com/gohugoio/hugo/issues/8239
-		// This is an error situation. We need something to vendor.
-		if t.Mounts() == nil {
-			return fmt.Errorf("cannot vendor module %q, need at least one mount", t.Path())
-		}
-
 		fmt.Fprintln(&modulesContent, "# "+t.Path()+" "+t.Version())
 
 		dir := t.Dir()
-- 
2.42.0

